instagram/src/App.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import './App.css';
+import dummyData from './dummy-data';
+import PostContainer from './components/PostContainer/PostContainer';
+import SearchBar from './components/SearchBar/SearchBar';
+// import SimpleStorage from "react-simple-storage";
+import Authenticate from './components/Authentication/Authentication';
+import LoginPage from './components/Login/Login';
+import PostsPage from './components/PostContainer/PostsPage';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      posts: dummyData,
+      filterInput: ''
+  }
+  }
+
+  // componentDidMount() {
+  //   // if (!localStorage.getItem('posts')) {
+  //     this.setState({
+  //       posts: dummyData
+  //     })
+  //   // }
+  //   // else {
+  //   //   console.log("Didn't work!")
+  //   // }
+  // }
+
+  handleChange = event => {
+    event.preventDefault();
+    this.setState({
+      inputText: event.target.value
+    })
+  }
+
+  filterComments = event => {
+    event.preventDefault();
+    this.setState({
+      filterInput: event.target.value,
```

Comment:
It's messing some sort of check to reset the value back. So if you changed the insides of this function to something like ```if (event.target.value === "") {
      this.setState({
        posts: dummyData
      });
    } else {
      this.setState({
        filterInput: event.target.value,
        posts: this.state.posts.filter(e =>
          e.username.includes(this.state.filterInput)
        )
      });
    }``` It would work again after you delete the search input.